src/mcts/node.cc

```diff
@@ -197,35 +249,51 @@ void Node::CreateEdges(const MoveList& moves) {
 Node::ConstIterator Node::Edges() const { return {edges_, &child_}; }
 Node::Iterator Node::Edges() { return {edges_, &child_}; }
 
+// Recalculate n_ from real children visits.
+// This is needed if a node was proved to be certain in a prior
+// search and later gets to be root of search.
+void Node::ReComputeN() {
```

Comment:
No need for capital `C` in a name. Something like `RecomputeNFromChildren` is even better.

Reply:
done